cdhweb/resources/admin.py

```diff
@@ -4,7 +4,8 @@
 from mezzanine.pages.models import Page, RichTextPage
```

Comment:
this file can get deleted too; its admin will all move to `pages`.

Reply:
addressed by 4a10cd49b355fd2fe6aabe0b62b7d63058af6625